### DISCUSSION TITLE
Erro ao tentar executar o blog.sql

### DISCUSSION THREAD

**Discussion by SHJordan** [balta-io/forum#435] [Tira Dúvidas] [answered]
Quando tentei executar a query  do arquivo usando sqlcmd (sqlcmd -S localhost,1433 -U sa -P 1q2w3e4r@#$ -d Blog -i blog.sql) pra popular o banco recebi o seguinte erro:
<details><summary>Details</summary>
<p>

sqlcmd -S localhost,1433 -U sa -P 1q2w3e4r@#$ -d Blog -i blog.sql

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)
Msg 547, Level 16, State 1, Server f069c93ced22, Line 32
The INSERT statement conflicted with the FOREIGN KEY constraint "FK_PostTag_TagI
d". The conflict occurred in database "Blog", table "dbo.Post", column 'Id'.
The statement has been terminated.
Msg 547, Level 16, State 1, Server f069c93ced22, Line 33
The INSERT statement conflicted with the FOREIGN KEY constraint "FK_PostTag_TagI
d". The conflict occurred in database "Blog", table "dbo.Post", column 'Id'.
The statement has been terminated.
Msg 547, Level 16, State 1, Server f069c93ced22, Line 34
The INSERT statement conflicted with the FOREIGN KEY constraint "FK_PostTag_TagI
d". The conflict occurred in database "Blog", table "dbo.Post", column 'Id'.
The statement has been terminated.

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

(1 rows affected)

Process finished with exit code 0.

</p>
</details>

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @SHJordan, este erro ocorre quando tenta inserir um PostTag (Vínculo de Post com Tag) na tabela mas não há um POST ou TAG com o ID. 

Você pode ignorar este erro, ou seguir linha a linha para ver qual registro em sí dá o erro e incluir um POST/TAG com os respectivos IDs antes.
